wallet/backend.go

```diff
@@ -0,0 +1,48 @@
+package wallet
+
+import (
+	"errors"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4/ecdsa"
+	"golang.org/x/crypto/blake2b"
+	"io"
+	"perun.network/go-perun/wallet"
+	"perun.network/perun-ckb-backend/wallet/address"
+)
+
+type backend struct {
+}
+
+var Backend = backend{}
+
+func init() {
+	wallet.SetBackend(Backend)
+}
+
+func (b backend) NewAddress() wallet.Address {
+	return &address.Address{}
+}
+
+func (b backend) DecodeSig(reader io.Reader) (wallet.Sig, error) {
```

Comment:
Add a docstring reinforcing that we only use padded signatures with this backend. The interface method itself is too ambiguous for our use-case and someone having to adapt this backend might miss out on this fact.

Reply:
Done!